layouts/partials/topbar.html

```diff
@@ -13,8 +13,8 @@
 
         <!-- Begin .a-globalNav -->
         <nav class="a-globalNav " id="primary-nav">
-          <a href="{{ with .Site.GetPage `/`}}{{.URL}}{{end}}" class="a-globalNav-logo" style="background-color:white; padding: 0.2em;">
-            <img style="vertical-align: bottom;" src="/images/AltinnD-logo.svg" alt="Altinn Studio logo" class="an-logo adocs-logo">
+          <a href="https://www.altinndigital.no/" class="a-globalNav-logo">
```

Comment:
I suggest dropping changing this page from this PR. Does not make sense as an open source project to have this header, needs more discussion.